app/assets/stylesheets/app/froggo_teams.scss

```diff
@@ -75,13 +75,37 @@
     font-family: $main-font;
   }
 
-  &__organization-title {
-    align-items: center;
+  &__organization-teams-section {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    padding: 15px;
+  }
+
+  &__teams-section {
+    flex: 1;
+    
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Unexpected whitespace at end of line   no-eol-whitespace